src/frontend/apps/desk/src/features/header/AccountDropdown.tsx

```diff
@@ -9,9 +9,13 @@ export const AccountDropdown = () => {
   const { t } = useTranslation();
   const { logout } = useAuthStore();
 
+  const [isDropOpen, setIsDropOpen] = useState(false);
+
   return (
     <DropButton
       aria-label={t('My account')}
+      onOpenChange={(isOpen) => setIsDropOpen(isOpen)}
+      isOpen={isDropOpen}
```

Comment:
If you active the focus on isLocalOpen in `DropButton`, you will not need all these changes.